### ISSUE TITLE
Migrate driver from UDBC-SQLite3 to Pharo-SQLite3

### ISSUE THREAD

**Issue by PierceNg** [pharo-rdbms/glorp-sqlite3#1]
As subject.

**Comment by eMaringolo** [comment]
I don't know who is the admin of the repo, but can we create an `udbc` tag, so we can continue committing in the master branch but using Pharo-SQLite3 instead of the UDBC version?

**Comment by astares** [comment]
There is no admin: it's a community repo, see https://github.com/orgs/pharo-rdbms/people

You can join and help if you like

**Comment by PierceNg** [comment]
Did the following: 
- Updated ```p7_newsqlite``` branch. This branch now works with Pharo-SQLite3 on Pharo 7. One blob test fails.

Plan to do the following:
- Update ```p7_udbc``` branch to load UDBC-SQLite3 from Github not Smalltalkhub.
- Test ```p7_newsqlite``` branch with Pharo 8.
- Make ```p7_newsqlite``` branch the new ```master``` upon successful testing with Pharo 8.

When done, ```p7_udbc``` will be the legacy UDBC version, tested with Pharo 7, and ```master``` will be use Pharo-SQLite3 going forward.

Thoughts?

**Comment by eMaringolo** [comment]
I think we should start using version tags, so in the future the dependency could be made on a particular tag, and that would give us the freedom to move forward without breaking references to the head (implicit) of a particular branch.

**Comment by astares** [comment]
Yes - I guess we could try in separate branches, but keep the master as "master" and tag it with releases. Similar to other projects. And I doubt people will have a problem migrating since the changes are straightforward.

**Comment by eMaringolo** [comment]
We can have an udbc branch from the last udbc based commit and let people know about it (just in case somebody wants to continue using it).
